### ISSUE TITLE
Fix broken links in UI

### ISSUE THREAD

**Issue by antonmc** [IBM-Bluemix/insurance-store-front#2]
The `About` and `Login` links currently return 404s.

**Comment by antonmc** [comment]
Need to define boundaries of this shrink wrapped version

**Comment by uwefassnacht** [comment]
I think the `about` should point to the github README.md

The `login`link should not be active (at least not at this point)

**Comment by jakepeyser** [comment]
I agree we need to link to the source code, but not in that manner. Footer is generally more appropriate as it is not part of the "experience"

**Comment by uwefassnacht** [comment]
Sure. Either way if fine with me. But then we need to deactivate the `about` link
